llama-demo-apps/VideoSummary.ipynb

```diff
@@ -0,0 +1,599 @@
+      "File \u001b[0;32m~/anaconda3/envs/llama-demo-apps/lib/python3.8/site-packages/langchain/llms/replicate.py:138\u001b[0m, in \u001b[0;36mReplicate._call\u001b[0;34m(self, prompt, stop, run_manager, **kwargs)\u001b[0m\n\u001b[1;32m    136\u001b[0m prediction\u001b[38;5;241m.\u001b[39mwait()\n\u001b[1;32m    137\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m prediction\u001b[38;5;241m.\u001b[39mstatus \u001b[38;5;241m==\u001b[39m \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mfailed\u001b[39m\u001b[38;5;124m\"\u001b[39m:\n\u001b[0;32m--> 138\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mRuntimeError\u001b[39;00m(prediction\u001b[38;5;241m.\u001b[39merror)\n\u001b[1;32m    139\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(prediction\u001b[38;5;241m.\u001b[39moutput, \u001b[38;5;28mstr\u001b[39m):\n\u001b[1;32m    140\u001b[0m     completion \u001b[38;5;241m=\u001b[39m prediction\u001b[38;5;241m.\u001b[39moutput\n",
+      "\u001b[0;31mRuntimeError\u001b[0m: Your input is too long. Max input length is 4096 tokens, but you supplied 5608 tokens."
+     ]
+    }
+   ],
+   "source": [
+    "chain = load_summarize_chain(llm, chain_type=\"refine\")\n",
```

Comment:
Shall we add context here on what `chain_type="refine"` used for?

Reply:
yes - added at the top.